openhtf/__init__.py

```diff
@@ -428,6 +428,14 @@ def wrap_or_copy(cls, func, **options):
     retval.options.update(**options)
     return retval
 
+  def _asdict(self):
```

Comment:
also need to remove `func` and add `name` and `doc` instead.  we could move this up a level though since these semantics are specific to the `station_api`?

i'd be okay putting the type-skipping in `convert_to_base_types` but that only really removes the `ignore_keys=('cls',)` bit.